### ISSUE TITLE
Notifications being received, but HomeKit not firing events.

### ISSUE THREAD

**Issue by hjdhjd** [hjdhjd/homebridge-doorbird#10]
Hi guys,

Glad to see we've shifted to the notification API (YAY for that!). It's reliable in my experience...but here's what's happening:

[6/15/2017, 8:10:26 AM] [Doorbell Platform] DoorBird resetting doorbell
[6/15/2017, 8:10:48 AM] [Doorbell Platform] HTTP Server: doorbell.html was called, sending event to homebridge.
[6/15/2017, 8:10:53 AM] [Doorbell Platform] DoorBird resetting doorbell
[6/15/2017, 8:11:05 AM] [Doorbell Platform] HTTP Server: doorbell.html was called, sending event to homebridge.

So...the events are being received by the HTTP server, but what isn't happening is I'm not getting a notification of the event on an iDevice. When I monitor it within the Home app...weirdly it shows that motion events were triggered with the doorbell (not the motion device...which I have disabled) -- not sure if that's a HomeKit quirk, or if something else is up there.

TLDR: Doorbird is firing events correctly and sending to the HTTP server. HTTP server doesn't seem to be properly triggering events back to HomeKit.

Thoughts?

**Comment by brownad** [comment]
You sure the motion sensor has "Allow Notifications" enabled post the upgrade?  If you're seeing the events flow through to iOS you're pretty much there...

**Comment by brownad** [comment]
I have to say its been flawless lately, given you can set the relaxation too on the API its even better :)

**Comment by mrhund** [comment]
Hi,

one thing I noticed during testing: When sending multiple events in a short periode of time homekit stopped sending the push notifications for some minutes. After waiting a couple of minutes everything workes fine again. Can you confirm this behavior?
Imho some type of flooding protection....

**Comment by brownad** [comment]
Possibly something on HomeKit as previously you could flood it to death

**Comment by brownad** [comment]
I noticed the same looking into it

**Comment by brownad** [comment]
Nothing that I can pin just yet seems another plugin was failing (harmony hub) but it's back and working now

**Comment by hjdhjd** [comment]
Still not seeing it. Next I'm going to try removing Doorbird from HomeKit and adding it back in to see if that helps. Very odd.

**Comment by brownad** [comment]
Closing this one
